client/src/js/services/exportFile.js

```diff
@@ -131,7 +134,7 @@ angular.module('bhima.services')
     e.click();
 	}
 
-	function csv(fileData, fileName, separator) {
+	function csv(fileData, fileName, genDate, separator) {
```

Comment:
@DedrickEnc, actually, the [JavaScript Object Notation (JSON) standard](http://json.org/example.html) uses `camelCase`instead of `under_scores`.  I'm not sure who the authors of [j-io](http://www.j-io.org/) are, but they are in the minority of JavaScript developers in this aspect.

Further resources: 
 1. [AngularJS's Style Guide](https://github.com/johnpapa/angular-styleguide#style-y230) recommends using the JSHint option "camelcase", which requires that all variables are either `camelCase` or `UPPER_CASE_WITH_UNDERSCORES`.  [Here is the JSHint reference'(http://jshint.com/docs/options/#camelcase).
 1. [AirBnB's JavaScript Styleguide](https://github.com/airbnb/javascript#22.2) also recommends camelCase for functions, object, and instances.
 1. [JQuery's style guide](https://contribute.jquery.org/style-guide/js/#naming-conventions) says that "Variable and function names should be full words, using camel case with a lowercase first letter. Names should be descriptive but not excessively so."
 1. Finally [Google's JavaScript Style Guide](https://google.github.io/styleguide/javascriptguide.xml)  says this:

> In general, use functionNamesLikeThis, variableNamesLikeThis, ClassNamesLikeThis, EnumNamesLikeThis, methodNamesLikeThis, CONSTANT_VALUES_LIKE_THIS, foo.namespaceNamesLikeThis.bar, and filenameslikethis.js.
>